3party/opening_hours/opening_hours_tests/opening_hours_tests.cpp

```diff
@@ -112,11 +112,37 @@ bool IsOpen(osmoh::TRuleSequences const & rules, std::string const & dateTime)
   return GetRulesState(rules, dateTime) == osmoh::RuleState::Open;
 }
 
+std::string GetNextTimeOpen(osmoh::TRuleSequences const & rules, char const * fmt, std::string const & dateTime)
+{
+  std::tm time = {};
```

Comment:
There is no need to initialize the variable. It will be overwritten below.

Reply:
Good catch!
The bigger problem is that there is no strptime on Windows… But it's not too important atm.